week2/Week2.py

```diff
@@ -0,0 +1,140 @@
+# Task 1 : Checking type of number
+
+def fizz_buzz(num):
+    if num % 3 == 0 and num % 5 == 0:
+        print(num, "-", "Fizz Buzz")
+    elif num == 1 or num == 2 or num == 3 or num == 5:
+        print(num, "-", "Prime")
+    elif num % 5 == 0 and num != 5:
+        print(num, "-", "Buzz")
+    elif num % 3 == 0 and num != 3:
+        print(num, "-", "Fizz")
+    elif num % 2 != 0:
+        for i in range(3, 100):
+            if (num % i) != 0:
+                print(num, "-", "Prime")
+            break
+    else:
+        print(num)
```

Comment:
The syntax looks good, but why all this extra for the "Prime" output? None of the conditions actually check for a prime number. I am confused 

Why can't there be a "Buzz", when `num !=5`?

Reply:
oops. I mixed things here. Sorry I modified it now.